lib/audit_test.pm

```diff
@@ -33,8 +33,9 @@ our @EXPORT = qw(
 );
 
 our $tmp_dir = '/tmp/';
-our $test_dir = '/usr/local/eal4_testing';
-our $code_repo = get_var('CODE_BASE', 'https://gitlab.suse.de/security/audit-test-sle15/-/archive/master/audit-test-sle15-master.tar');
+our $test_dir = get_var('IPSEC_TEST') ? '/usr/local/ipsec' : '/usr/local/eal4_testing';
+our $default_code_base = get_var('IPSEC_TEST') ? 'https://gitlab.suse.de/QA-APAC-I/ipsec/-/archive/main/ipsec-main.tar' : 'https://gitlab.suse.de/security/audit-test-sle15/-/archive/master/audit-test-sle15-master.tar';
```

Comment:
Personal question:
https://gitlab.suse.de/QA-APAC-I/ipsec/-/archive/main/ipsec-main.tar
We should use this git lab link in future, corret

Reply:
yes, I think so. If we move this test code to other place, we could define `CODE_BASE` in job settings to change it.

Reply:
Then no problem